bigquery/src/main/java/com/google/cloud/bigquery/samples/AsyncQuerySample.java

```diff
@@ -41,23 +38,20 @@
    * @throws IOException IOException
    * @throws InterruptedException InterruptedException
    */
-  public static void main(final String[] args)
-      throws IOException, InterruptedException {
+  public static void main(final String[] args) throws IOException, InterruptedException {
     Scanner scanner = new Scanner(System.in);
```

Comment:
That was the way it was in the existing sample, and it didn't bother me as much as some of the other things I fixed. I can look at changing this, too.